apps/functions/register-representation/register-representation-command/index.js

```diff
@@ -0,0 +1,59 @@
+import api from './back-office-api-client.js';
+import { isEmpty, pick } from 'lodash-es';
+
+const mapContactDetails = (entity) => {
+	if (isEmpty(entity)) return {};
+
+	return {
+		...pick(entity, [
+			'firstName',
+			'lastName',
+			'organisationName',
+			'jobTitle',
+			'email',
+			'phoneNumber',
+			'type',
+			'under18'
+		]),
+		...(!isEmpty(entity.address)
+			? {
+					address: pick(entity.address, [
+						'addressLine1',
+						'addressLine2',
+						'town',
+						'county',
+						'postcode',
+						'country'
+					])
+			  }
+			: undefined)
+	};
+};
+
+/**
+ *
+ * @param {import('@azure/functions').Context} context
+ * @param {import('@pins/api/src/message-schemas/commands/register-representation').RegisterRepresentation} msg
+ */
+export default async function (context, msg) {
+	context.log(`Handle register-representation event: ${JSON.stringify(msg)}`);
```

Comment:
Should we only log the ID for PII reasons?

Reply:
Yep quite right, this was for debugging so no longer needed. Thanks!